app/src/main/java/io/apicurio/registry/rest/v2/GroupsResourceImpl.java

```diff
@@ -594,6 +624,23 @@ private static final void requireParameter(String parameterName, Object paramete
         }
     }
 
+    private static void maxOneOf(String parameterOneName, Object parameterOneValue, String parameterTwoName, Object parameterTwoValue) {
+        if (parameterOneValue != null && parameterTwoValue != null) {
+            throw new ConflictException("Both " + parameterOneName + " and " + parameterTwoName + " are present.");
```

Comment:
Let's change the language here to:  "Conflict: '" + parameterOneName + "' and '" + parameterTwoName + "' are mutually exclusive."

Reply:
This exception should not be used here. This ConflictException is being used in the compatibility layer and is subject to change there and should not be used in the rest layer. You can use one of the existing ones or you think this is worth a new one just create it.